\documentclass{article}
\usepackage{url}
\usepackage{breakurl}
%\VignetteIndexEntry{The EPICv2manifest package user's guide}
%\VignetteEngine{knitr::knitr}
\begin{document}
\title{The \texttt{EPICv2manifest} package user's guide}

\author{Peters TJ, Pidsley R}
\maketitle

\section*{Package Contents}

\texttt{EPICv2manifest} is an annotation package providing a data.frame object containing the complete 49 column Infinium Illumina EPIC v2.0 probe manifest (``IlmnID'' constitutes the rownames), plus 31 additional columns derived from Peters et al. (2024). 

<<libr, message=FALSE, warning=FALSE>>=
library(EPICv2manifest)
@

\section*{Additional columns}

\begin{itemize}

\item \verb!CpG_chrm!, \verb!CpG_beg!, \verb!CpG_end! - EPICv2 coordinates from Sesame manifest 

\item \texttt{MismatchPos} - vector of probes where ``Y'' indicates discrepant genomic position between Sesame and Illumina, includes those that are missing mapping information in Illumina manifest (chr0) 

\item \texttt{MissingPos} - vector of probes where ``Y'' indicates probes missing mapping information in Illumina manifest 

\item \texttt{namerep} - vector of probes where ``Y'' indicates probes that have replicates based on probe name match (this vector can be married up with \texttt{Name} column to select a single probe to represent each name replicate) 

\item \texttt{seqrep} - vector of probes where ``Y'' indicates probes that have exact sequence matches with other probes within EPICv2 

\item \verb!seqrep_IlmnIDs! - IlmnIDs of those probes with exact sequence matches with other probes within EPICv2 (corresponding to \texttt{seqrep}) 

\item \verb!seqrep_RepNum! - replicate number of those probes with exact sequence matches with other probes within EPICv2 (corresponding to \texttt{seqrep} and \verb!seqrep_IlmnIDs!). Can be used to e.g. filter for only 1 probe per sequence-replicate. 

\item \texttt{posrep} - vector of probes where ``Y'' indicates probes that have exact genomic position matches with other probes within EPICv2 (based on Illumina genomic positions – note, none of the affected probes are discrepant with Sesame mapping) 

\item \verb!posrep_IlmnIDs! - IlmnIDs of those probes with exact genomic position matches with other probes within EPICv2 (corresponding to \texttt{posrep}) 

\item \verb!posrep_RepNum! - replicate number of those probes with exact genomic position matches with other probes within EPICv2 (corresponding to \texttt{posrep} and \verb!posrep_IlmnIDs!). Can be used to e.g. filter for only 1 probe per genomic-position-replicate. 

\item \texttt{EPICv1probeID} - vector of EPICv1 probe names where probe names match between EPICv1 and EPICv2 

\item \texttt{EPICv1seqmatch} - vector of EPICv1 probe names where probe sequences match between EPICv1 and EPICv2 

\item \texttt{EPICv1locmatch} - vector of EPICv1 probe names where genomic locations match between EPICv1 and EPICv2 (based on Sesame locations) 

\item \texttt{K450probeID} - vector of 450K probe names where probe names match between 450K and EPICv2 

\item \texttt{K450seqmatch} - vector of 450K probe names where probe sequences match between 450K and EPICv2 

\item \texttt{K450locmatch} - vector of 450K probe names where genomic locations match between 450K and EPICv2 (based on Sesame locations) 

\item \texttt{K450locmatch2} - vector of 450K probe names where additional 450K probes have a genomic location match between 450K and EPICv2 (based on Sesame locations) 

\item \texttt{K27probeID} - vector of 27K probe names where probe names match between 27K and EPICv2 

\item \texttt{K27seqmatch} - vector of 27K probe names where probe sequences match between 27K and EPICv2 

\item \texttt{K27locmatch} - vector of 27K probe names where genomic locations match between 27K and EPICv2 (based on Sesame locations) 

\item \texttt{K27locmatch2} - vector of 27K probe names where additional 27K probes have a genomic location match between 27K and EPICv2 (based on Sesame locations) 

\item \verb!CH_BLAT! - vector of probes where ``Y'' indicates at least one in silico cross-hybridisation event ($\geq 47$ bp match) to a non-target region of the genome, predicted by BLAT (Kent 2002) 

\item \verb!CH_WGBS_evidence! – subset vector of \verb!CH_BLAT! where ``Y'' indicates a greater affinity for the off-target(s), via comparison to whole genome bisulphite sequencing on matched samples 

\item \verb!RMSE_with_WGBS! – root mean squared error when comparing probe methylation to matched target CpG site methylation from WGBS (\textit{M}-values) 

\item \verb!Num_offtargets! – number of off-target \textit{in silico} hybridisation events predicted by the probe sequence via BLAT (Kent 2002)

\item \verb!Suggested_offtarget! - if \verb!CH_WGBS_evidence! == ``Y'', the hg38 coordinate of the off-target cytosine conferring minimum RMSE with WGBS 

\item \verb!Rep_results_by_NAME! – results of competitive comparison between replicates, with replicate probe sets defined by column \texttt{Name}

\item \verb!Rep_results_by_SEQUENCE! - results of competitive comparison between replicates, with replicate probe sets defined by column \verb!seqrep_IlmnIDs!

\item \verb!Rep_results_by_LOCATION! -  results of competitive comparison between replicates, with replicate probe sets defined by column \verb!posrep_IlmnIDs!

\end{itemize}

\section*{Sources}


\begin{thebibliography}{9}

Kent, W. J. (2002). BLAT--the BLAST-like alignment tool. Genome Research, \textbf{12}(4), 656–664. 


\end{thebibliography}

\end{document}
